Tests/SnapshotTests.swift

```diff
@@ -121,7 +121,8 @@ class SnapshotTests: XCTestCase {
         assertSnapshot(
           matching: animationView,
           as: .imageOfPresentationLayer(
-            precision: SnapshotConfiguration.forSample(named: sampleAnimationName).precision),
+            precision: SnapshotConfiguration.forSample(named: sampleAnimationName).precision,
+            perceptualPrecision: 0.985),
```

Comment:
I was getting diff failures locally on master, probably because I'm using a different Xcode version than CI. I enabled `perceptualPrecision` (a new-ish feature in the snapshotting library) which should prevent that sort of issue from now on.